### PR TITLE
Fixed problem with assert failing

### PR DESCRIPTION
**Pull request by HonokaNo**
Fixed problem with assert failing in test_2 and test_3 functions.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -10,12 +10,12 @@ void test_1(void)
 
 void test_2(void)
 {
-    assert(1 + 1 == 3);
+    assert(1 + 1 == 2);
```
Comment (katahiromz) [inline main.cpp:14]: ここにわかりやすくコメントを追加。

File: main.cpp [hunk 2]
```diff
@@ -10,12 +10,12 @@ void test_1(void)
...
 }
 
 void test_3(void)
 {
-    assert(2 + 3 == 6);
+    assert(2 + 3 == 5);
```
Comment (katahiromz) [inline main.cpp:20]: ここにも。

**Comment by HonokaNo** [comment]
I have added a comment regarding the above

**Comment by HonokaNo** [comment]
Fixed.

File: main.cpp (hunk 1 above)
Comment (HonokaNo) [reply main.cpp:14]: Fixed.

File: main.cpp (hunk 2 above)
Comment (HonokaNo) [reply main.cpp:20]: Fixed.

**Pull request merged**